src/test/java/com/wikia/webdriver/elements/mercury/old/TableOfContentPageObject.java

```diff
@@ -25,7 +25,7 @@
   private WebElement tocButton;
   @FindBy(xpath = "//section[contains(@class, 'article-body')]/article[contains(@class, 'article-content')]/*[1]")
   private WebElement tocAtTheTop;
-  @FindBy(xpath = "//section[contains(@class, 'article-body')]/article[contains(@class, 'article-content')]/aside[contains(@class, 'portable-infobox')][position() = 1]/following-sibling::*[1]")
+  @FindBy(xpath = "//section[contains(@class, 'article-body')]/article[contains(@class, 'article-content')]/aside[contains(@class, 'portable-infobox')][position() = 1]/following-sibling::*[2]")
```

Comment:
What do you think about leaving a comment that we want second following sibling of infobox to be a ToC because first one is an ad?